introduction.tex

```diff
@@ -25,7 +26,9 @@ \chapter{Introduction}
 construct fully autonomous systems, critical and less critical systems must be able to share
 resources. However, resource sharing between components of mixed-criticality is heavily restricted by current
 standards.
-
+% Non-fungibility of time,
+% Difficulty in calculating wcet of high critical tasks -> overbooking.
+% Varying processor time requirements of all tasks -> overbooking.
```

Comment:
I guess you decided to not include explaining these here and include them later in the chapter.  But it might be helpful to use a sentence to explain why mcs systems can explode when one task overruns its budget and why that is hard to fix/ leads to massive overestimation in budgets of critical tasks.